test/integration/appDuplicate.spec.ts

```diff
@@ -10,10 +10,18 @@ test('duplicate app from home flow', async ({ page }) => {
   await homeModel.goto();
   const app = await homeModel.createApplication({ dom });
   await homeModel.goto();
-  await homeModel.duplicateApplication(app.name);
+  const duplicateApp = await homeModel.duplicateApplication(app.name);
   await homeModel.goto();
   await expect(page.locator(locators.toolpadHomeAppRow(`${app.name} (copy)`))).toBeVisible();
   await homeModel.duplicateApplication(app.name);
   await homeModel.goto();
```

Comment:
maybe no need to check twice? unless there's something specific about the logic we're trying to cover here

Reply:
Trying to check if the second duplicate is appropriately named

Reply:
got it, makes sense then!